usb.c

```diff
@@ -260,7 +267,25 @@ int qdl_write(struct qdl_device *qdl, const void *buf, size_t len)
 	return count;
 }
 
-void qdl_set_out_chunk_size(struct qdl_device *qdl, long size)
+static void usb_set_out_chunk_size(struct qdl_device *qdl, long size)
 {
-	qdl->out_chunk_size = size;
+	struct qdl_device_usb *qdl_usb = (struct qdl_device_usb *) qdl;
+
+	qdl_usb->out_chunk_size = size;
 }
+
+int usb_init(struct qdl_device **qdl)
```

Comment:
As we don't care to differentiate between errors, please just return the struct qdl_device *, or NULL on error.

Reply:
fixed, thanks for the suggestion!